packages/suite/src/components/wallet/AccountsMenu/components/AccountSearchBox/index.tsx

```diff
@@ -96,22 +95,31 @@ const AccountSearchBox = (props: Props) => {
                 clearButton
                 onClear={onClear}
             />
-            <CoinsFilter>
+            <CoinsFilter
+                onClick={() => {
+                    setCoinFilter(undefined);
+                }}
+            >
                 {props.networks.map(n => (
-                    <OuterCircle key={n} isMobile={props.isMobile} isSelected={coinFilter === n}>
+                    <OuterCircle
+                        key={n}
+                        isMobile={props.isMobile}
+                        isSelected={coinFilter === n}
+                        onClick={e => {
+                            e.stopPropagation();
+                            const alreadySelected = coinFilter === n;
+                            if (alreadySelected) {
+                                setCoinFilter(undefined);
+                            } else {
+                                setCoinFilter(n);
+                            }
```

Comment:
Could be simplified to `setCoinFilter(coinFilter === n ? undefined : n)`-

Reply:
I am not fully convinced it helps readability (aka what it is the intention of the code), but I guess that's problem for coders that will come after us 😁

Reply:
updated, I also added a comment to prevent said confusion